packages/commons/src/monitoring.ts

```diff
@@ -0,0 +1,16 @@
+import pkgUp = require('pkg-up')
+
+async function init() {
+  if (process.env.NODE_ENV !== 'production') {
+    const path = await pkgUp()
+
+    require('elastic-apm-node').start({
+      // Override service name from package.json
+      // Allowed characters: a-z, A-Z, 0-9, -, _, and space
+      serviceName: require(path!).name.replace('@', '').replace('/', '_'),
```

Comment:
This defines the service name to be whatever is in the package's  "name" field in package.json